LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2022 Bytedance Ltd. and/or its affiliates.
```

Comment:
@BlockLiu  
This is the License header template that needs to be applied to the source code.
We are not supposed to add the copyright information here.
Could you review the change?

Reply:
Just LICENSE file right?

Reply:
Fixed, see https://github.com/bytedance/bitsail/pull/342.
Thanks for your comment!